### ISSUE TITLE
Two optimizers

### ISSUE THREAD

**Issue by Rebeccazizi** [intuitive-robots/flower_vla_calvin#3]
Paper said “FLOWER deploys separate optimizers with
distinct learning rates for the Flow Transformer and the VLM. The VLM optimizer leverages a lower
warm-up rate to prevent overshooting”，is this part of the code open source?

**Comment by mbreuss** [comment]
Hi, the dual optimizer stuff is in the pretraining code, that I plan to release soon. After pretraining the model  it's not really needed anymore for fine-tuning, thus for simplicity I removed the dual stuff in here.
